Client/Ecosia/UI/PageAction/PageActionMenu.swift

```diff
@@ -166,20 +166,15 @@ extension PageActionMenu: UITableViewDataSource, UITableViewDelegate {
         return cell
     }
 
-    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
-        return UIView()
-    }
-
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        if section == 0 {
-            return UITableView.automaticDimension
-        } else {
-            return UX.spacing
```

Comment:
Is `UX.spacing` used anywhere else? Or can it be removed from that struct then?

Reply:
Sure thing! @lucaschifino ✅